### ISSUE TITLE
Machine ID: Modify docs to use Machine ID helm chart

### ISSUE THREAD

**Issue by strideynet** [gravitational/teleport#46633]
## Applies To

- https://goteleport.com/docs/enroll-resources/machine-id/deployment/kubernetes/
- https://goteleport.com/docs/admin-guides/access-controls/access-request-plugins/ssh-approval-slack/

## Details

Following https://github.com/gravitational/teleport/pull/44615 and https://github.com/gravitational/teleport/pull/46353, we now have a Machine ID helm chart that can integrate with some of the access plugin charts. We should update the Machine ID and access plugin documentation to reflect this.

**Comment by stevenGravy** [comment]
Another option is using a Kubernetes job/cronjob that requires a machine-id access for say a database access or external k8s call. This would be a good example to have.

**Comment by zmb3** [comment]
@strideynet do we still believe this is the direction we want to go?

**Comment by strideynet** [comment]
> @strideynet do we still believe this is the direction we want to go?

Yes - I think this would still be valuable. It's been on my list to pick up sometime this month for at least the MWI guides.
